com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/activiti/Messages.java

```diff
@@ -7,6 +7,7 @@
     public static final String ABORT_OPERATION_TIMED_OUT = "Abort operation timed out";
     public static final String ACTIVITI_JOB_RETRY_FAILED = "Activiti job retry failed";
     public static final String PROCESS_STEP_NOT_REACHED_BEFORE_TIMEOUT = "Step \"{0}\" of process \"{1}\" not reached before timeout";
+    public static final String NO_VALUE_GIVEN_FOR_REQUIRED_PARAMETERS = "No value given for one or more required parameters {0}";
```

Comment:
Now that I think about this a bit more, I think we should change it to `"Service \"{0}\" has missing required parameters: {1}"`. That way, the user would know **exactly** where to look in his deployment descriptor.